examples/backtrace_crate/src/main.rs

```diff
@@ -0,0 +1,14 @@
+extern crate backtrace;
```

Comment:
As discussed, I don't think this example adds much beyond the panic backtrace example. Consider removing